Capri.Services/ServiceResult.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Capri.Web.ViewModels;
+
+namespace Capri.Services
+{
+    public class ServiceResult<T> : Dictionary<string, string>
+    {
+        public T TBody { get; private set; }
+        public bool IsSucceded { get; private set; }
```

Comment:
It will be much easier for you to use such implementation:
```

        private bool? _successful;
        public bool Successful => _successful ?? IsValid;
        public bool IsValid => !this.Any();

        public ServiceResult()
        {
        }

        public ServiceResult(bool successful)
        {
            _successful = successful;
        }
```

Right now IsSucceded is binded to errors inside. You don't have to remember to set that explicitly.
Also if you would like to create explicitly serviceResult with some status, then you can use such constructor. ;)